contracts/babylon/src/contract.rs

```diff
@@ -216,6 +224,7 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> Result<QueryResponse, Cont
             &queries::babylon_checkpoint(deps, epoch_number)?,
         )?),
         QueryMsg::CzLastHeader {} => Ok(to_json_binary(&queries::cz_last_header(deps)?)?),
+        QueryMsg::CzLastHeight {} => Ok(to_json_binary(&queries::cz_last_height(deps)?)?),
```

Comment:
Why can't we reuse `CzLastHeader`, e.g., let this API return `None` if no header is found?

Reply:
This is just an improvement for performance. It's also more robust, as zero is used to mean "not found" / not set. Finally, this query is simpler to transform into a raw query (again for performance).